frontend/hub/execution-environments/hooks/useExecutionEnvironments.tsx

```diff
@@ -0,0 +1,10 @@
+import { useGet } from '../../../common/crud/useGet';
+import { HubItemsResponse } from '../../useHubView';
+import { ExecutionEnvironment } from '../ExecutionEnvironment';
+
+export function useExecutionEnvironments() {
+  const t = useGet<HubItemsResponse<ExecutionEnvironment>>(
+    '/api/automation-hub/v3/plugin/execution-environments/repositories/'
```

Comment:
```suggestion
    process.env.HUB_API_BASE_PATH + '/v3/plugin/execution-environments/repositories/'
```

(if #606 gets merged first)